src/erlang.zig

```diff
@@ -3,10 +3,10 @@ pub const ei = @cImport({
 });
 
 const std = @import("std");
-pub const receiver = @import("receiver.zig");
-pub const sender = @import("sender.zig");
+pub const Reader = @import("Reader.zig");
+pub const writer = @import("writer.zig");
```

Comment:
Just a comment: I'm not certain the new names are better. This may be bias on my end though.

Reply:
I did consider using `encoder` and `decoder`, but I don't have a particularly strong opinion. What do you think of those?

Reply:
I prefer those (`encoder` and `Decoder`), but not strongly. If you have the time and patience, change it, otherwise proceed as it is.